### ISSUE TITLE
Help with installing...

### ISSUE THREAD

**Issue by Mattjdlewis** [mrbungle64/ecovacs-deebot.js#108]
### The question

Hi I am new to github and this hole world of amazing things, I have a deebot t8 AIVI and im trying to find the coordinates for a custom clean. I noticed you wrote on another page that you can acquire it by downloading your node.js?

can you help me by giving me an explanation to how i download it and use it? I am completely new to all this so if you could make the instructions easy for me so i can learn how to do this. sorry if this is written in the wrong place i just not sure at this time how to do it all.

Kind regards

**Comment by mrbungle64** [comment]
@Mattjdlewis 

Do I understand correctly that you need support installing [Node.js](https://nodejs.org/en/)?

**Comment by Mattjdlewis** [comment]
yes thats correct, thanks for getting back so quickly

**Comment by mrbungle64** [comment]
@Mattjdlewis 

For information on installing Node.js for all possible operating systems, see the following links:

* https://nodejs.org/en/download/
* https://nodejs.org/en/download/package-manager/
* https://github.com/nodesource/distributions/blob/master/README.md

But I wonder what your actual goal is in using my library?

**Comment by mrbungle64** [comment]
@Mattjdlewis 

Yes, you can find out the coordinates by following these instructions:
https://github.com/mrbungle64/ecovacs-deebot.js/wiki/Retrieve-coordinates-from-log-output#customarea

If you are you using Deebot for Home Assistant you can also find the coordinates in the logs of Home Assistant:

"Use the app to send the vacuum to a custom area and afterwards search your logs for Last custom area values (x1,y1,x2,y2): entries to get the coordinates"
https://github.com/And3rsL/Deebot-for-Home-Assistant#example-commands

**Comment by mrbungle64** [comment]
> I've gone to HA - configuration - logs but it does not show any logs for deebot, could it be that im looking in the wrong place?

How to activate the logs is described in this section:
https://github.com/And3rsL/Deebot-for-Home-Assistant#issues

**Comment by mrbungle64** [comment]
> then i sent it to do a custom_clean in the mobile app, went back to logs but nothing is there?

Is this a question?

> when i click show LOAD FULL HA LOG im getting an error.

What does the error message look like?

> if i then refresh HA it has a connection error and i have to go back to nabu casa to connect?

Is this a question?
